src/Notifier.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Beaver\Dateval;
+
+class Notifier extends \PHPMailer {
+    
+    private $event,
```

Comment:
__[code_sniffer]__
* There must not be more than one property declared per statement